collapsiblecalendarview2/src/main/res/values/strings.xml

```diff
@@ -8,4 +8,7 @@
     <string name="friday">FRI</string>
     <string name="saturday">SAT</string>
     <string name="sunday">SUN</string>
+    <string name="content_description_eiv_arrow">expand or collapse calendar</string>
```

Comment:
This should be split into "Expand calendar button" and "Collapse calendar button" and then programmatically changed. Generally speaking, we need to satisfy [name, role, and value criteria](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html).